var/spack/repos/builtin/packages/callpath/package.py

```diff
@@ -46,7 +46,9 @@ def install(self, spec, prefix):
         # TODO: offer options for the walker used.
         cmake_args = std_cmake_args
         if spec.satisfies("^dyninst@9.3.0:"):
-            cmake_args.append("-DCMAKE_CXX_FLAGS='-std=c++11 -fpermissive'")
+            std_flag = self.compiler.cxx11_flag
+            cmake_args.append("-DCMAKE_CXX_FLAGS='{0} -fpermissive'".format(
+                    std_flag))
```

Comment:
Flake8 doesn't like this line:
```
var/spack/repos/builtin/packages/callpath/package.py:51: [E126] continuation line over-indented for hanging indent
```
It should be indented 4 spaces more than the previous line.